org.pitivi.Pitivi.json

```diff
@@ -55,16 +55,29 @@
             "no-autodownload": true
         }
     },
-    "modules" : [
+    "modules": [
         "python3-matplotlib.json",
         {
-            "name" : "gsound",
-            "sources" : [
+            "name": "libcanberra",
```

Comment:
We want to get rid of gsound but until we can, libcanberra is required by gsound.